api/system-settings/getOtpToken

```diff
@@ -33,8 +32,7 @@ my $input = NethServer::ApiTools::readInput();
 my $username = $input->{'username'} || '';
 my $OtpStatus;
 my $key;
-my $domainName = $db->get_value('DomainName');
-my $systemName = $db->get_value('SystemName');
+my $hostname = `/usr/bin/hostname`;
```

Comment:
Do not run an external command to obtain the host name. Instead

- ask e-smith DB (like you did in lines L36-37)
- use the Perl API (see some examples here https://github.com/search?l=Perl&q=Sys%3A%3AHostname+org%3Anethserver&type=Code)

Reply:
e-smith it is not possible because I do not want to use the `sudo`, however thank for your second proposal it works